jenkins/test_history/requirements.txt

```diff
@@ -1,3 +0,0 @@
-requests
```

Comment:
deleting this file breaks gen_history, since it installs these requirements inside a docker container